addons/storyshots/storyshots-core/src/frameworks/config-loader.js

```diff
@@ -1,43 +0,0 @@
-import fs from 'fs';
-import path from 'path';
-import { getBabelConfig } from '@storybook/core/server';
```

Comment:
This export can probably be removed from `@storybook/core` now, as well as `@storybook/<framework>/options.js` endpoints

Reply:
Will remove